engine/include/cubos/engine/gizmos/gizmos.hpp

```diff
@@ -0,0 +1,114 @@
+/// @file
+/// @brief Resource @ref cubos::engine::Gizmos.
+/// @ingroup gizmos-plugin
+
+#pragma once
+#include <vector>
+
+#include <glm/glm.hpp>
```

Comment:
```suggestion
#include <glm/vec3.hpp>
```

At least in this header you don't need anything else. Might have to change the source file includes.